app/views/docs/getting-started-for-server.phtml

```diff
@@ -39,6 +39,24 @@
             <pre class="line-numbers"><code class="prism language-typescript" data-prism>import * as sdk from "https://deno.land/x/appwrite/mod.ts";</code></pre>
         </div>
     </li>
+    <li>
```

Comment:
To be consistent with other language, we might better use a command line installation over manual dependency setup. Shouldn't `pub get dart_appwrite` be enough?

Importing the file is already covered in the next example, we can skip it.

Reply:
There is no way to do it, otherwise I would have done that, `pub get <package_name>` doesn't work. Only way is first to add to pubspec.yaml file then run `pub get`